sdkproject/Assets/Mapbox/Unity/MeshGeneration/Modifiers/MeshModifiers/ChamferHeightModifier.cs

```diff
@@ -0,0 +1,391 @@
+namespace Mapbox.Unity.MeshGeneration.Modifiers
+{
+	using System.Collections.Generic;
+	using UnityEngine;
+	using Mapbox.Unity.MeshGeneration.Data;
+
+	[CreateAssetMenu(menuName = "Mapbox/Modifiers/Smooth Height for Buildings Modifier")]
+	public class ChamferHeightModifier : MeshModifier
+	{
+		[SerializeField]
+		private bool _flatTops;
```

Comment:
if you don't check flat tops, the roof is not positioned correctly, relative to the walls.